### PR TITLE
Use BSL license

### PR DESCRIPTION
**Pull request by ryanwjackson**

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1,21 +1,97 @@
...
+                        third parties.
+
+Notice
+
+The Business Source License (this document, or the “License”) is not an Open
+Source license. However, the Licensed Work will eventually be made available
+under an Open Source License, as stated in this License.
+
+License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
+“Business Source License” is a trademark of MariaDB Corporation Ab.
```
Comment (SeanBolt) [inline LICENSE.txt:26]: @ryanwjackson There are a few references to MariaDB in here, do we want to change them to ModernTreasury?

File: LICENSE.txt [hunk 2]
```diff
@@ -1,21 +1,97 @@
...
+Licensor or its affiliates (provided that you may use a trademark or logo of
+Licensor as expressly required by this License).
+
+TO THE EXTENT PERMITTED BY APPLICABLE LAW, THE LICENSED WORK IS PROVIDED ON
+AN “AS IS” BASIS. LICENSOR HEREBY DISCLAIMS ALL WARRANTIES AND CONDITIONS,
+EXPRESS OR IMPLIED, INCLUDING (WITHOUT LIMITATION) WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, NON-INFRINGEMENT, AND
+TITLE.
+
+MariaDB hereby grants you permission to use this License’s text to license
```
Comment (SeanBolt) [inline LICENSE.txt:74]: Mentioned here

File: LICENSE.txt [hunk 3]
```diff
@@ -1,21 +1,97 @@
...
+your works, and to refer to it using the trademark “Business Source License”,
+as long as you comply with the Covenants of Licensor below.
+
+Covenants of Licensor
+
+In consideration of the right to use this License’s text and the “Business
+Source License” name and trademark, Licensor covenants to MariaDB, and to all
```
Comment (SeanBolt) [inline LICENSE.txt:81]: Mentioned here

File: LICENSE.txt (hunk 1 above)
Comment (ryanwjackson) [reply LICENSE.txt:26]: @SeanBolt this License and TM is the copyright of MariaDB.  We must include this part in order to use it.

**Pull request merged**